Mobile form

HFG43 opened the pull request:
Hi, thanks for your feedback!
I have corrected as requested errors in the textarea placeholder style.
To ensure changes apply to any browser, I have added code to also run on IE 10+, Edge apart from others, should you use any of these options.

Hi there, many thanks for your previous feedback! Very much appreciated!
I'm sending the PR with the suggestions and required changes.

Hi there!

I'm sending this PR for you to review my code. Thank you in advance for your feedback! 🙌🙌
For this PR I have added the following:
* Work on Pair Programming
* Add a contact-me section with a form
* Ensure client-side validation (for this case using HTML)
* Add a Formspree action form to collect all messages and user data
* Ensure no Linter errors
* Update README file

HFG43 committed:
Contact Me Section in place

Style and text set

HFG43 committed:
Add section Margin

HFG43 committed:
Set form style & skeleton

HFG43 committed:
Review Contact Me BG position

HFG43 committed:
Style form inputs

Borders and padding

HFG43 committed:
Contact button padding

Extend horizontal padding

M-Anwar-Hussaini committed:
Add HTML form validations on the contact-me section.

HFG43 committed:
Form Textarea message validation

HFG43 committed:
Update README file

HFG43 committed:
Review Readme

Capitalize description

HFG43 committed:
Contact Form BG in place

KanzaTahreem commented on style.css:
```diff
@@ -269,3 +270,68 @@ html {
+  outline: none;
+  padding: 15px 16px;
+}
+
+.textarea-size {
+  height: 9.375rem;
+}
+
```
- [ ] I noticed that the recommended `font-family` is not applied on the `textarea's placeholder`. You can [check out this link](https://redstapler.co/how-to-change-placeholder-font-css/) to learn how you can do that 💪🛠 Similarly, please use the given `color` as well. You can learn about changing placeholder's color from [here](https://www.w3schools.com/howto/howto_css_placeholder.asp) ℹ

Your design | Figma design 
---| ---| 
![image](https://user-images.githubusercontent.com/106119105/235757789-544ce0c4-f8f0-42d7-8c52-6f8ae9f5ea64.png) | ![image](https://user-images.githubusercontent.com/106119105/235757723-b5b32023-34fe-4ef6-9a66-b37871b4201c.png)

KanzaTahreem commented on index.html:
```diff
@@ -1,156 +1,328 @@
+            placeholder="Write your message here."
+            required
+          ></textarea>
+          <button type="submit" class="see-project-button contact-form-button">
+            Get in touch
+          </button>
```
- [ ] I noticed that you are using `border-radius: 5px` . Please change it to `8px` as given in the Figma design guidelines 🤓⛏
- [ ] Similarly, the color given in the design guidelines is `#6070FF` not `#396df2`. Please fix this as well 💪
- [ ] For `hover` effect, please consider adding `box-shadow` also as given.
- [OPTIONAL] I think it would be better to use `cursor: pointer` when  a user interacts with the button. What do you think? 💭😅

KanzaTahreem reviewed (changes requested):
# Changes Required 🔁

Hi @HFG43 & @M-Anwar-Hussaini 👋

Well done with the project so far 🥇 I really appreciate the effort you put in!  **_However, there are few things that still need to be fixed But you are almost there!_** 💪 🏁

![so_close](https://user-images.githubusercontent.com/99715640/211022680-440a1bb1-4281-4acb-8dbc-3442fe509063.gif)

## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Well done following the given Figma design
- ✅ Great job with CSS
- ✅ Descriptive PR and README file
- ✅ Correct Linters

However, there is always room for improvement.

## Required Changes 🛑

> _Please keep in mind that your long hours of work is appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄_

_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please check the comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HFG43 committed:
CR changes

Remove "previous content" from README file
Correct input max chars
Justify text in contact form paragraph
Change font weight & color in textarea placeholder
Correct Button design

HFG43 committed:
Correct Linter

GedeonTS commented on index.html:
```diff
@@ -1,156 +1,328 @@
+            name="user-email"
+            placeholder="hernanguemes@hotmail.com"
+            required
+          />
+          <textarea
+            name="message"
+            id="message"
```
- Kindly make Sure to follow your last Reviewer Feedback [here](https://github.com/HFG43/Portfolio_Mobile_First/pull/3#discussion_r1182946021) and stick to the Figma design just as requested. I think it would be better to change the placeholder text.

**Your Design**
![235757789-544ce0c4-f8f0-42d7-8c52-6f8ae9f5ea64](https://user-images.githubusercontent.com/97834160/235774767-a86e5e77-171f-47ff-bf9f-e6938fa9541c.png)
**Figma Design**
![235757723-b5b32023-34fe-4ef6-9a66-b37871b4201c](https://user-images.githubusercontent.com/97834160/235774807-5b6b4755-ce4c-424e-9ec1-b184ec234588.png)

HFG43 replied:
@GedeonTS Hi! Many thanks for your feedback. Please let me know if I can please call you through zoom one moment, if that's fine for you

GedeonTS reviewed (changes requested):
# STATUS: CHANGES REQUIRED :recycle: :
Hi @HFG43 and @M-Anwar-Hussaini , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

<img src="https://media.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif">

 ## Highlights :tophat: :+1: 
- Good PR Description :+1: 
- Linters are passing :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HFG43 committed:
Textarea Placeholder

HFG43 committed:
Update Placeholder Style

Efiamotu-1 commented on index.html:
[OPTIONAL]
- [ ] please kindly use descriptive words as placeholders in your contact forms so that users of your portfolio can be exactly sure of what to imput in a agiving fiels. see the design template below as example reference :+1: 
![exam](https://user-images.githubusercontent.com/89055502/235815016-e853e953-370e-4816-9013-08d059ea1c37.png)

HFG43 replied:
Thanks @Efiamotu-1 ; I have included your suggestions!

Efiamotu-1 reviewed (approved):
Hi @HFG43 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

![giphy](https://user-images.githubusercontent.com/89055502/234114221-4ca295de-fd8f-4af6-9c49-a6d4c6ea67f7.gif)


## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [![@Efiamotu-1](https://img.shields.io/github/followers/Efiamotu-1?label=Habeeb-Efiamotu-Musa&style=social)](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[![Habeeb Efiamotu Musa](https://img.shields.io/badge/linkedin-@Habeeb-crimson)](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [![@EFYAMOTU](https://img.shields.io/twitter/follow/omarramoun?label=Habeeb-Efiamotu-Musa&style=social)](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [![Habeeb Efiamotu Musa ](https://img.shields.io/badge/slack-@Habeeb-brightgreen)](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HFG43 committed:
Add more descriptive placeholders

As suggested in the CR, I changed the placeholders to be more descriptive
